qemu/tests/cpu_device_hotplug.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import re
+import time
+
+from autotest.client.shared import error
+from autotest.client.shared import utils
+
+from virttest import utils_misc
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    Runs vCPU hotplug tests based on CPU device:
+    """
+    def get_cpu_model():
```

Comment:
Using this function can get the cpu model. I added this for future use. For testing different variations of cpu models, namely
POWER8E_v2.1,  POWER8E,   POWER8_v2.0,  POWER8,   POWER7_v2.3 , POWER7  ,POWER8NVL_v1.0,  POWER8NVL
Currently experimenting with the above by adding in the configuration file.